src/DicomWebClient/API/IDicomWebClient.cs

```diff
@@ -28,5 +29,36 @@ public interface IDicomWebClient
         /// Provides DICOMweb WADO services for retrieving studies, series, instances, frames and bulkdata.
         /// </summary>
         IWadoService Wado { get; }
+
+        /// <summary>
+        /// Provides DICOMweb QIDO services for querying a remote server for studies.
+        /// </summary>
+        IQidoService Qido { get; }
+
+        /// <summary>
+        /// Provides DICOMweb STOW services for storing DICOM instances.
+        /// </summary>
+        IStowService Stow { get; }
+
+        /// <summary>
+        /// Configures the service URI of the DICOMweb service.
+        /// </summary>
+        /// <param name="uriRoot">Base URL of the DICOMweb server.</param>
+        /// <param name="wadoUrlPrefix">Optional URL path prefix for WADO RESTful services.</param>
+        /// <param name="qidoUrlPrefix">Optional URL path prefix for QIDO RESTful services.</param>
+        /// <param name="stowUrlPrefix">Optional URL path prefix for STOW RESTful services.</param>
+        /// <param name="deleteUrlPrefix">Optional URL path prefix for DELETE RESTful services.></param>
+        void ConfigureServiceUris(
+            Uri uriRoot,
```

Comment:
Nit picking on the parameter name. In the DICOM Web context, the URI is really a URL, and the scheme is limited to http(s), so it is better to call it url (though the .net core class itself is Uri for the lack of Url).

Reply:
Since it's a Uri type.  As other params they are `url`.

Reply:
if http: or https: are required, the `Uri` class supports validation for that.